### ISSUE TITLE
Creating Topic Problem

### ISSUE THREAD

**Issue by hogiro** [mittwald/typo3-forum#110]
Hello,

I could install the Forum and can see the forums. As I try to create a topic I get an error and after it even the create Topic Button wont be shown anymore. 

That is the Error code:
Oops, an error occurred! Code: 201602190547580418d911 

There is another jQuery error on Forum-Pages as following:

SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
...trim(b);if(a.JSON&&a.JSON.parse)return a.JSON.parse(b);if(n.test(b.replace(o,"@"...

I hope someone knows a solution.

Thanks

**Comment by velletti** [comment]
I think you have to look to the SysLog and search for the Error Code with that ID 201602190547580418d911 (this is just the timestamp of the error)  
To get the "real" PHP Error message. Something was wrong during storing the New Topic. 

You can also : delete all entries in  Topics and Posts, Reset the fields Last Post, Last Topics, Topic Count in torum table ( or delete that (hopefully test Forum) , then Enable Display Errors in Instalatool to see the real PHP Error and not only the timestamp.

The reason for the missing Create Button: is the Broken Topic/Post Data: 
There is a Json Call (with Fire Bug or Chrome Developer Tools you can see it unter network -> XHR) 

The Request goes to the ajax Controller and shloud deliver a json Resopnse. BUT this seemed now also be broken. 

If You need further assistance, just ask ..

**Comment by hogiro** [comment]
Hello veletti,

Thanks for your Help. I already fixed the problem with creating the Topic problem and did even a pull request. Your information was very helpful for me.

unfortunately I still have the problem with the new Topic/Post Button. After creating the topic the button wont be shown anymore. Just if I clear the cache the button will be visible again. I will try to fix this problem too but if you have any information for me, I would appreciate it very much.

Best Regards,

**Comment by alat54** [comment]
Same issue for the button which desappears...

Thanks

**Comment by mirkokaufmann** [comment]
It was a caching problem. Please have a look at current develop branch. Buttons will now be loaded via ajax to be able to cache forum list view and topic show view.

**Comment by hogiro** [comment]
I just tested it, but unfortunately I cannot see any buttons, not important what I did, I couldnt see any buttons.

**Comment by hogiro** [comment]
Unfortunately I still didnt get any answer. After Update I dont see any buttons and not important what I configured I see always that the Topics are locked.

**Comment by velletti** [comment]
@hogiro 
if you can give me access to the installation, i can have a look what is missing in you case ...
 
If you are interested ( this first help is free of charge) send me your contactdata, the link to backend and Frontend  password to typo3 [at] velletti.de I can have a look durng the next evenings ..

**Comment by hogiro** [comment]
Hi velletti,

unfortunately it is a local installation. Even yesterday I installed all things new, a fresh typo3 installation without any other extensions and just installed typo3_forum from github. The result is the same. Before the update at least I could see the buttons after I cleaned the cache. So I think it has to do with the update.

**Comment by hogiro** [comment]
sorry but the problem still occurs.

**Comment by pstranghoener** [comment]
@hogiro Did you install our latest version from the master branch? https://github.com/mittwald/typo3-forum/tree/1.0.1 And followed the installation documentation? The new version was release at April 7.

**Comment by hogiro** [comment]
Hello Philipp,
yesterday I wanted to test all again. I installed a clean new Typo3 v 7.6.6 and used the typo3_forum 1.0.2 from the TER. Unfortunately I still cannot see any buttons. As I said, the installation is clean and I included the Templates and set the Forum id.

What I realized is, that I cannot see the Forum Plugin by folder options for Contains Plugin. In earlier Versions I had it in Dropdown Box and could select it.

Wann bist du bei Slack Online? Dann könnte ich dir dort die Zugangsdaten mitteilen und wir könnten das Problem gern zusammen durchgehen.

Thanks
Devrim
